### PR TITLE
E2e/fix-flakiness

### PR DESCRIPTION
**Pull request by jamil314**

### CODE REVIEW

**Comment by rikukissa** [comment]
Your environment is deployed to https://e2efix-flakiness.opencrvs.dev.

File: e2e/testcases/correction-birth/correct-birth-record-7.spec.ts [hunk 1]
```diff
@@ -261,13 +256,18 @@ test.describe.serial(' Correct record - 7', () => {
        */
 
       await expect(
-        page.locator('#mother-content #Date').getByRole('deletion')
-      ).toHaveText(formatDateTo_ddMMMMyyyy(declaration.mother.birthDate))
+        page.locator('#mother-content #Age').getByRole('deletion')
+      ).toHaveText(
+        joinValuesWith(
+          [declaration.mother.ageOfIndividualInYears, 'years'],
+          ' '
```
Comment (makelicious) [inline e2e/testcases/correction-birth/correct-birth-record-7.spec.ts:263]: ' ' is default, but explicitly is ok

File: e2e/testcases/death/helpers.ts [hunk 2]
```diff
@@ -30,11 +35,11 @@ export type DeathDeclarationInput = {
...
   deceased: {
     name: {
-      firstNames: faker.name.firstName('male'),
-      familyName: faker.name.lastName('male')
+      firstNames: faker.name.firstName('male') + generateRandomSuffix(),
+      familyName: faker.name.lastName('male') + generateRandomSuffix()
     },
     gender: 'male',
-    birthDate: getRandomDate(75, 200),
+    age: random(20, 100),
```
Comment (makelicious) [inline e2e/testcases/death/helpers.ts:42]: Let's revisit this if there are too many clashes. Then we just need to keep track of the values

File: e2e/helpers.ts [hunk 3]
```diff
@@ -323,7 +323,27 @@ export const drawSignature = async (page: Page) => {
   }
 }
 
-export const expectOutboxToBeEmpty = async (page: Page) =>
+export const expectOutboxToBeEmpty = async (page: Page) => {
+  /*
+   * This is to ensure the following condition is asserted
+   * after the outbox has the declaration
+   */
+  await page.waitForTimeout(500)
```
Comment (makelicious) [inline e2e/helpers.ts]: Let's use constants we created

**Review by makelicious — APPROVED** [review]
👍

**Pull request merged**